cassandra/metadata.py

```diff
@@ -475,14 +475,16 @@ def make_token_replica_map(self, token_to_host_owner, ring):
         ring_len_range = range(ring_len)
```

Comment:
I guessed it was vestigial from an earlier implementation of the algorithm. I can change. I thought I did (as I also noticed it), but must have backed it out in my own rework.